README.md

```diff
@@ -182,64 +191,134 @@ require("jupynium").setup({
+3. Save your buffer as `<filename>.ju.py`
+
+When using Jupynium for the first time, it's recommended to start a new notebook to make sure everything works before trying to load existing files.
+
+### Connect to the Jupynium server
+
+**This is for local Neovim only. For remote Neovim, see [Command-Line Usage](#%EF%B8%8F-command-line-usage-attach-to-remote-neovim).**
+
+The Jupynium server stays alive as long as the browser is alive. So you can see them as the same thing in this doc.
 For example:
 
 - Starting Jupynium server = opening a Selenium browser
 - Manually closing the browser = closing the Jupynium server
 
-### Open and attach to a Jupynium server
+In Neovim, with your Jupynium `.ju.py` file open, you can run `:JupyniumStartAndAttachToServer` to start the notebook server.
 
-**This is for local neovim only. For remote neovim, see [Command-Line Usage](#%EF%B8%8F-command-line-usage-attach-to-remote-neovim).**
+#### Sync current buffer to the Jupynium server
 
-Running `:JupyniumStartAndAttachToServer` will open the notebook.  
-Type password and once **you need to be on the main page (file browser) for the next steps**.
+You need to be on the main notebook page (file browser) for the next few steps.
 
-**New in 0.1.1:**  
-Jupynium will open Jupyter Notebook server for you if not found.  
-It will also open the ipynb file in the current directory and ask you if you want to sync from vim or from ipynb.
+Although Neovim is now attached to the server, it won't automatically start syncing.
 
-### Sync current buffer to the Jupynium server
-
-You attached your nvim instance to the server, but it won't automatically start syncing.  
-Run `:JupyniumStartSync` to create a new ipynb file and start syncing from vim to notebook.
+To sync your Neovim Jupynium file to a notebook, run `:JupyniumStartSync`.
 
 You can also:
 
-1. `:JupyniumStartSync filename` to give name (`filename.ipynb`) instead of `Untitled.ipynb`.
+- `:JupyniumStartSync filename` to give a name to the notebook (`filename.ipynb`) instead of `Untitled.ipynb`. This does not open existing files. If a file with that name already exists then the filename argument will just be ignored.
+- To sync a Jupynium file to an existing notebook, manually open the file in the browser, and `:JupyniumStartSync 2` to sync to the 2nd tab (count from 1).
 
-- This will not open a file if it exists. Instead filename will be ignored.
+At this point, any changes you make within the Neovim Jupynium file will be reflected live in the browser. Make sure you do not make changes inside the browser itself, as the sync is only one-way (from Neovim to browser).
 
-2. Manually open the file from the browser, and `:JupyniumStartSync 2` to sync using the 2nd tab (count from 1).
+If you want to save a copy of the `.ipynb` file, run `:JupyniumSaveIpynb`. There is also a configuration option to enable automatic saving/downloading.
```

Comment:
`:JupyniumSaveIpynb` saves the notebook file from the notebook server. It will also be done automatically when you save the vim file and there's no option to turn it off yet.

If you want to keep a copy you need to use `:JupyniumDownloadIpynb`.